src/org/labkey/test/tests/wiki/WikiTest.java

```diff
@@ -198,6 +202,32 @@ public void testShowPageTreeForWiki()
         assertElementPresent(Locator.id("wiki-toc-tree").append(Locator.linkContainingText(WIKI_PAGE_TITLE + " (" + WIKI_PAGE_NAME + ")")));
     }
 
+    /*
+        Regression coverage for
+        https://www.labkey.org/home/Developer/issues/Secure/issues-details.view?issueId=48019
+
+     */
+    @Test
+    public void testWikiWithComma()
+    {
+        String wikiName = "Wiki with comma's";
+        String wikiTitle = "Comma in the content";
+        String wikiContent = "This is my HTML, with commas";
+
+        goToProjectHome();
+        log("Creating the wiki " + wikiTitle);
+        WikiHelper wikiHelper = new WikiHelper(this);
+        wikiHelper.createNewWikiPage("HTML");
+        numberOfWikiCreated++;
+        setFormElement(Locator.name("name"), wikiName);
+        setFormElement(Locator.name("title"), wikiTitle);
+        wikiHelper.setWikiBody(wikiContent);
```

Comment:
If I remember correctly, the wiki helper doesn't know how to use the graphical editor. It switches to the html source tab automatically. If so, this needs to have some html tags added explicitly.
```suggestion
        wikiHelper.setWikiBody("<p>" + wikiContent + "</p>");
```